src/i18n/strings/en_EN.json

```diff
@@ -2179,6 +2174,26 @@
             "one": "Currently removing messages in %(count)s room",
             "other": "Currently removing messages in %(count)s rooms"
         },
+        "release_announcement": {
+            "done": "Done",
+            "filter": {
+                "description": "Filter your chats with a single click. Expand to view more filters.",
+                "title": "New quick filters"
+            },
+            "intro": {
+                "description": "The chats list has been updated to be more clear and simple to use.",
+                "title": "Chats has a new look!"
+            },
+            "next": "Next",
+            "settings": {
+                "description": "To show or hide message previews, go to All settings > General > Room list",
```

Comment:
This is wrong, the settings are under the preferences of the user settings:

<img width="1080" height="876" alt="image" src="https://github.com/user-attachments/assets/d3505806-013f-43a9-b9c3-3476e18973e1" />

Reply:
Good point - symptom of settings not using latest designs yet. Fixed.